Quadriga/src/main/java/edu/asu/spring/quadriga/domain/projectblog/IProjectBlogEntry.java

```diff
@@ -38,4 +38,10 @@
 
     IUser getAuthor();
 
+    /**
+     * @param wordCount
+     *            , number of words needed in the snippet
+     * 
+     */
+    String getSnippet(int wordCount);
```

Comment:
This needs real javadoc. What is this method for, what does it do? Does it create a summary or just cut the first n words, etc.